.pre-commit-config.yaml

```
@@ -2,6 +2,13 @@
 minimum_pre_commit_version: 4.0.0
 default_install_hook_types: [pre-commit, commit-msg]
 
+ci:
+  autofix_commit_msg: |
+    Apply automatic fixes from pre-commit hooks
+  autofix_prs: true
+  autoupdate_commit_msg: 'Bump pre-commit hooks'
```

I think we should use conventional commits here as well.
```suggestion
  autoupdate_commit_msg: 'ci: Bump pre-commit hooks'
```

---

Thanks @peterdragun for the review and suggestion!